test/unit/client/product_client/payment/create_test.js

```diff
@@ -47,7 +47,7 @@ describe('products client - creating a new payment', () => {
       response = productFixtures.validCreatePaymentResponse({product_external_id: productExternalId})
       provider.addInteraction(
         new PactInteractionBuilder(`${PRODUCTS_RESOURCE}/${productExternalId}/payments`)
-          .withUponReceiving('a valid create charge create request')
+          .withUponReceiving('a valid create charge create request 1')
```

Comment:
I guess ` 1` in the end of the string was added for temporary debugging?

Reply:
Yep, it was failing some Pact tests because of the reuse of reference, but now I should remove it.